common/faceting.js

```diff
@@ -452,12 +437,39 @@
                         }
                     }
 
+                    /**
+                     * Given an object with `date` and `time`, it will turn it into timestamp reprsentation of datetime.
+                     * @param  {object} obj The datetime object with `date` and `time` attributes
+                     * @return {string} timestamp in submission format
+                     * NOTE might return `null`
+                     */
+                    function dateTimeToTimestamp(obj) {
+                        if (!obj) return null;
```

Comment:
Having this guard is good because it protects against calling `absMin.date` when absMin is `undefined` or `null`.